static/src/javascripts/projects/common/modules/commercial/dfp/dfp-api.js

```diff
@@ -118,49 +119,64 @@ define([
     var prebidService        = null;
     var googletag;
 
-    var callbacks = {
-        '300,251': function (event, $adSlot) {
-            stickyMpu($adSlot);
-        },
-        '300,250': function (event, $adSlot) {
-            if (config.switches.viewability && $adSlot.hasClass('ad-slot--right')) {
-                var mobileAdSizes = $adSlot.attr('data-mobile');
-                if (mobileAdSizes && mobileAdSizes.indexOf('300,251') > -1) {
-                    stickyMpu($adSlot);
-                }
-            }
-        },
-        '1,1': function (event, $adSlot) {
-            if (!event.slot.getOutOfPage()) {
-                $adSlot.addClass('u-h');
-                var $parent = $adSlot.parent();
-                // if in a slice, add the 'no mpu' class
-                if ($parent.hasClass('js-fc-slice-mpu-candidate')) {
-                    $parent.addClass('fc-slice__item--no-mpu');
-                }
-            }
-        },
-        '300,1050': function () {
-            // remove geo most popular
-            geoMostPopular.whenRendered.then(function (geoMostPopular) {
-                fastdom.write(function () {
-                    bonzo(geoMostPopular.elem).remove();
-                });
-            });
-        }
-    };
     var renderStartTime = null;
-    var recordFirstAdRendered = once(function () {
-        beacon.beaconCounts('ad-render');
-    });
     var prebidEnabled = config.switches.headerBiddingUs && config.page.edition === 'US';
 
+    var windowResize = debounce(
```

Comment:
Unfortunately, that will run afoul of eslint's `no-use-before-define` restriction, because `postDisplay` passes the var to a callback. Any suggestions?